diesel/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "diesel"
-version = "2.1.0"
+version = "2.1.1"
```

Comment:
shouldn't that be part of a separate pr/commit?

Reply:
```suggestion
version = "2.1.0"
```